src/platform-includes/getting-started-primer/kotlin-multiplatform.mdx

```diff
@@ -1,13 +1,10 @@
 Sentry's Kotlin Multiplatform SDK builds on top of multiple Sentry SDKs, allowing developers to use the same codebase and share code between platforms while being able to integrate Sentry's features into their applications.
 
-## Overview of Features
-
-- Native crash reporting for Android and JVM, leveraging our [Android SDK](/platforms/android) and [Java SDK](/platforms/java)
-- Native crash reporting for iOS, macOS, tvOS, and watchOS, leveraging our [Cocoa SDK](/platforms/apple)
-- Automatic breadcrumbs for app lifecycle and UI events
-
 <Note>
```

Comment:
`m`: this note is placed in slightly different positions on all 3 pages where it's used - I'd suggest to always put it below the install code snippet